package.json

```diff
@@ -16,31 +16,28 @@
     "curl",
     "debug"
   ],
-  "author": "delirius325",
+  "author": "Abhishek Rajeshirke <https://github.com/colidarity>",
   "contributors": [
     {
-      "name": "Justin \"jayceekay\"",
-      "url": "https://github.com/jayceekay"
-    },
-    {
-      "name": "Peter \"binggg\" Zhao",
-      "url": "https://github.com/binggg"
+      "name": "Abhishek Rajeshirke",
```

Comment:
Should be reverted to the other contributors, and keep yourself there of course 😄 !